notebooks/src/covid-19.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>The docstring for <code>noisy_sma_infections</code>  shouldn't be the same as for <code>sma_infections</code></p><p> </p><p>but actually -- my guess is that we shouldn't have <code>noisy_sma_infections</code>.</p><p> </p><p>I'm guessing the reason to have it is that otherwise, there are only <code># of scenarios in the Columbia model * number of percentiles available from the model</code>  possible peaks? Or maybe fewer, perhaps as few as only 1?</p><p> </p><p>If so, then I think we should fuzz the model results, not <code>sma_infections</code></p><p> </p><p>(BTW I'm realizing that this notebook is a reasonably rough draft, which TBC you had already told me over Slack. So from here I'm going to stop commenting on things like the docstring and focus instead on the overall organization. I'll plan to do another pass later for things like docstrings once you've had time to clean up the notebook.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>